packages/cw-orch-core/src/environment/state.rs

```diff
@@ -14,7 +19,7 @@ pub trait ChainState {
 }
 
 /// This Interface allows for managing the local state of a deployment on any CosmWasm-supported environment.
-pub trait StateInterface: Clone {
+pub trait StateInterface {
```

Comment:
We also can't keep `Clone` for StateInterface

Reply:
We can if we keep file lock in a global hashmap.

Reply:
left it with hashset, see https://github.com/AbstractSDK/cw-orchestrator/pull/326#discussion_r1595177270